x-pack/plugins/ml/public/jobs/jobs_list/components/job_filter_bar/job_filter_bar.js

```diff
@@ -30,7 +31,14 @@ function loadGroups() {
         value: g.id,
         view: (
           <div className="group-item">
-            <JobGroup name={g.id} /> <span>({g.jobIds.length} job{(g.jobIds.length === 1) ? '' : 's'})</span>
+            <JobGroup name={g.id} />
```

Comment:
@peteharverson not sure if the space in the original code between the `JobGroup` tag and opening `span` was there on purpose but I think so? If yes, the new line break will remove that space and would need to be enforced by something like `&nbsp;` or similar.

Reply:
Yes, the content of the `<JobGroup>` element has changed here - the space before the `<span>` has gone, plus the parentheses.

Before:
![image_before](https://user-images.githubusercontent.com/7405507/48627233-f42bf100-e9ab-11e8-86f3-cadb4fe3393c.png)

After:
![image_after](https://user-images.githubusercontent.com/7405507/48627240-faba6880-e9ab-11e8-8b22-c0b9727bcd99.png)

Reply:
fixed